src/styles/utils/variables.scss

```diff
@@ -0,0 +1,5 @@
+$daySize: 100px;
+$countDays: 31;
+$gapValue: 1px;
+$calendarWidth: $daySize * 7 + 10;
```

Comment:
The calculation for `$calendarWidth` seems incorrect. The expression `$daySize * 7 + 10` does not account for the gaps between the days. If you intend to include gaps, you should multiply the gap value by the number of gaps (which is 6 for a 7-day week) and add it to the total width.